tests/array.rs

```diff
@@ -1891,3 +1891,66 @@ fn array_macros() {
     let empty2: Array<f32, Ix2> = array![[]];
     assert_eq!(empty2, array![[]]);
 }
+
+#[test]
+fn test_as_contiguous() {
```

Comment:
Can you break this in separate test functions? It makes it easier to see what is broken (i.e. you run all the tests instead of stopping at the first failing one).

Reply:
I've split that large test function into more granular test functions under new module.